disnake/interactions/application_command.py

```diff
@@ -33,15 +33,18 @@
 MISSING = utils.MISSING
 
 if TYPE_CHECKING:
+    from ..client import Client
     from ..ext.commands import InvokableApplicationCommand
     from ..state import ConnectionState
     from ..types.interactions import (
         ApplicationCommandInteraction as ApplicationCommandInteractionPayload,
         ApplicationCommandInteractionData as ApplicationCommandInteractionDataPayload,
     )
 
+BotT = TypeVar("BotT", bound="Client")
```

Comment:
might be easier to just import `BotT` from the base module instead of redeclaring it